app/controllers/admin/event_schedule_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Admin
+  class EventScheduleController < Admin::BaseController
+    load_and_authorize_resource :event_schedule
+
+    def create
+      event_schedule = EventSchedule.create(get_event_schedule_params(params))
```

Comment:
Shouldn't be necessary,  load_and_authorize_resource :event_schedule should do that!